mitodl/mitxpro
# Added seed data command

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #37 

#### What's this PR do?
Adds a command to seed course data or deleted already-seeded course data

#### How should this be manually tested?
`manage.py seed_data`
`manage.py seed_data --delete`

- Check the course models in Django admin (`/admin/courses/`) to confirm that the objects were created
- Make sure the command output makes sense
- Make sure the commands are idempotent

#### Any background context you want to provide?
I added the migration for the sake of the course admin models (I hadn't noticed that you couldn't add a Course there without a Program in admin)
